src/Containers/Profile/profileContainer.jsx

```diff
@@ -128,7 +143,7 @@ export class ProfileContainer extends Component {
               placeholder=""
               onType={this.onKeyChange}
             />
-            <button className="button button-register">Edit</button>
+            <button className="button button-register ">Edit</button>
```

Comment:
@giseleiradu, why not using our `<Button />` component? I think it has the implementation of the spinner on it. What you need to do is only to trigger when to start loading and when to stop.

Reply:
I had already a button, and no one was ready to onboard me on that button. I kept mine because it also works and we are running out of time.

Reply:
It is simple, I think you can still do it. And if you need help, we are all there...